pg_search/src/index/directory.rs

```diff
@@ -148,7 +148,7 @@ impl WriterDirectory {
     /// It's also important to note that this functionis called in contexts that cannot
```

Comment:
this is annoying but could you fix the typo above here too

Reply:
Done.

GitHub allows creating suggestions as a comment, for instant merging.  ;)